__init__.py

```diff
@@ -12,24 +12,24 @@ class BoilStepWithCountdownReminders(StepBase):
     BoilStep with reminders that are set relative to end of boil
     '''
     REMINDER_NAMES = [
-        "Hop 1 Addition",
-        "Hop 2 Addition",
-        "Hop 3 Addition",
-        "Hop 4 Addition",
-        "Hop 5 Addition",
-        "Prepare yeast",
-        "Prepare finings",
-        "Prepare cooling system",
-        "Dose finings",
-        "Start hot-side cooling loop",
-        "Prepare fermenter"
+        "Hop 1 Addition                 as minutes boiltime to the end",
```

Comment:
I tested this and the admin looks good on desktop and mobile, but it doesn't quite make sense when the alerts themselves pop up, e.g.:

![image](https://user-images.githubusercontent.com/1311012/45534986-416ed380-b840-11e8-9c01-bfa2be7d6e18.png)

I would suggest splitting the text `" as minutes boiltime to the end"` from the `REMINDER_NAMES`, and adding it back in where the properties are declared, e.g.:

1) declare a variable `NAME_EPILOGUE = as minutes boiltime to the end`, 
2) make sure `reminder_00` uses it, e.g. :

```
reminder_00 = Property.Number(REMINDER_NAMES[0] + NAME_EPILOGUE, configurable=True ...
```
3) and update each of the other reminders accordingly.

Reply:
You are right. Did not take into account that the GUI is used in a mobile too. 
Maybe just using the info should be clear enough. 
 Your suggestion is fine too. 
Thanks for building this add on.